notebooks/Analyze_Text.ipynb

```diff
@@ -0,0 +1,2895 @@
+{
```

Comment:
<p>Maybe wrap the char_span repr in quotes to be easier to pick out?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/98/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
<p>Good idea. Done.</p>